rpa-builder/6.0/modules/ROOT/pages/workbench-toolbar-overview.adoc

```diff
@@ -0,0 +1,213 @@
+== Start Quick Creation Mode
+
+Use this function to speed up the initial Workflow creation. When you start a new Workflow project from scratch, click *Start Quick Creation Mode* (image:quick-creation-icon.png[The Start Quick Creation Mode icon, 2%, 2%]) to open the Quick Creation Mode interface.
+
+image:quick-creation-mode.png[The quick creation mode window, 30%, 30%]
+
+Depending on the state and position your Workflow project is in, Quick Creation Mode provides the Workflow objects that you can use. For every object you insert, the wizard automatically starts so you can configure the different steps without ever losing focus on the actual application you want to measure. You can also insert any possible object at any position later on.
+
+If, for example, you want to add a Window wait Action Step in front of an Image search in a Transaction you have already created, place the blue position indicator where you want to add the Action Step and then click Window wait. You can position the objects with the blue arrow buttons or by clicking on the desired position directly.
+
+The blue arrow buttons also enable you to collapse or expand the selected Workflows, Transactions, or Control Blocks to optimize the graphical representation of your Workflow.
+
+If the Quick Creation Mode interface is blocking parts of the application you need to use in the Workflow, click *Switch Monitor Side* (image:switch-monitor-side-icon.png[The Switch Monitor Side icon, 2%, 2%]) and the interface switches from the right to the left side of the screen and vice versa. After you finish creating the Workflow, close Quick Creation Mode by clicking *Exit Quick Creation Mode* (image:exit-quick-creation-mode.png[The Exit Quick Creation Mode icon, 2%, 2%]). After you exit Quick Creation Mode, the standard RPA Builder GUI shows again including the Workflow you just created.
+
+[[validate]]
+== Validate Workflow
+
+To check if a Workflow is ready for script generation, press *Validation* (image:validate-icon.png[The Validate Workflow icon, 2%, 2%]). If the project contains Workflow objects that are not yet configured or wrongly configured, errors, RPA Builder shows warnings or general messages as a list in the Validation result window. Double-click an entry in this list to navigate to the corresponding Workflow object.
+
+[[run-workflow]]
+== Run Workflow
+
+If you want to test the created Workflow, click *Run Workflow* (image:play-icon.png[The Run Workflow icon, 2%, 2%]) to generate, or update, the script file from the currently active project. The script executes automatically after its creation.
+
+When you start a Workflow from within RPA Builder, the workflow runs in design mode so you can already test whether the Workflow behaves as expected or still needs modifications before the production phase.
+The STC Guard module does not execute if an error occurs. Thus no windows are closed. The Clear screen Action Step does not execute in design mode either.
```

Comment:
Suggestion to tie this sentence in with the next one:

The STC Guard module does not execute in design mode if an error occurs.